system/apps_featured/117_gabuino/web/v1/examples.js

```diff
@@ -1142,3 +1142,33 @@ examples =
     '    return 0;\n' +
     '}\n'
 }
+
+document.getElementById('input-file')
+      .addEventListener('change', getFile)
+
+    function getFile(event) {
+	    const input = event.target
+      if ('files' in input && input.files.length > 0) {
+	      placeFileContent(
+          html_editor,
+          input.files[0])
+
+          <!-- dirty hack for now just to remove the drop down menu -->
```

Comment:
no no no, you are mixing javascript code and html code. `<!--` is a comment for HTML, not for javascript, replace it by `// dirt hack...`

Reply:
fixed and tested locally